packages/explorer/src/components/Navigation.tsx

```diff
@@ -14,13 +14,15 @@ function NavigationLink({ href, children }: { href: string; children: React.Reac
   const pathname = usePathname();
   const getLinkUrl = useWorldUrl();
   return (
-    <Link
-      href={getLinkUrl(href)}
-      className={cn("text-sm uppercase underline-offset-[16px]", {
-        "font-semibold underline decoration-orange-500 decoration-4": pathname === getLinkUrl(href),
-      })}
-    >
-      {children}
+    <Link href={getLinkUrl(href)} className="group px-3 text-sm font-semibold uppercase">
```

Comment:
fyi the nav is now semibold for all links rather than the current/active link, which I think helps the nav stand out relative to other text on the page and the active state is already communicated by the bottom border

Reply:
just checked it out, looks very nice! also should have added the properly sized clickable area right from the start, thanks for doing this